runc/tests/test.bats

```diff
@@ -0,0 +1,25 @@
+@test "Version matches" {
+    # Ensure that `pkg_version` from the plan (embedded in the package
+    # identifier) matches the output from `runc --version`.
+    version="$(echo ${PKGIDENT} | cut -d/ -f3)"
+    result="$(hab pkg exec "${PKGIDENT}" runc --version | head -n1)"
+    [ "$result" = "runc version ${version}" ]
+}
+
+@test "Binary is statically compiled" {
+    # The output of `file` will contain (among other data) the
+    # string "statically linked" or "dynamically linked"; we want the former.
+    result="$(hab pkg exec core/file file /hab/pkgs/"${PKGIDENT}"/bin/runc)"
+    [[ "$result" =~ "statically linked" ]]
+}
+
+@test "Compiled with libseccomp support" {
+    # There may be better ways of determining this, but a version with
+    # libseccomp support will include numerous strings pointing to the
+    # Go seccomp library; one that hasn't been so compiled will *not*
+    # include these, but will instead include references to indicate
+    # seccomp is not supported.
+    result="$(hab pkg exec core/binutils strings /hab/pkgs/${PKGIDENT}/bin/runc)"
+    [[ "$result" =~ "github.com/opencontainers/runc/vendor/github.com/seccomp/libseccomp-golang" ]]
+    ! [[ "$result" =~ "github.com/opencontainers/runc/libcontainer/seccomp/seccomp_unsupported.go" ]]
+}
```

Comment:
I'm OK with this test for now as I like seeing tests that make sure we are building with the support we think we have. .  This is a bit brittle because it might break on an upgrade if they move files around.  Perhaps in the long run we can work with upstream on a "verbose" version command that prints what is supported and what isn't.

Reply:
Yeah, I'm not super enthusiastic about this test implementation either. While it's OK for now, I'll see if I can find a simple-ish `runc` command to run that would only work if it was compiled with seccomp support.